### DISCUSSION TITLE
A recurring phrase

### DISCUSSION THREAD

**Discussion by TorinoSM** [kaixxx/noScribe#104] [Q&A] [answered]
Hi, I transcribed a 1.5 hour long video, the transcription quality was excellent, but after about 40 minutes from the beginning of the file, the single sentence began to repeat again and again as a result, and so on until the end of the file, although of course there was live speech, not only this sentence.
What can I do to analyze and fix this problem?

**Comment by kaixxx** [comment, marked as answer]
This is a known issue with the AI model that I use for the transcription. The latest version 0.5 of noScribe, which came out back in June, improved it a bit. But I cannot eliminate the problem alltogether, it may happen from time to time. 
To fix the problem:
- Open the transcript in noScribe Editor.
- Navigate the cursor to the position where the repeating text starts.
- In the bottom right corner, you'll see the exact timestamps (begin - end) of the current segment. Note down the end of the segment.
- Now go back to noScribe and restart the transcription from the end of the segment with the repeating text (enter the timestamp in the "Start" field). Monitor the output to make sure it is not repeating again (if this happens, start a little later in the audio). 
- Afterward, you must join the two transcripts manually.

**Reply by TorinoSM** [reply]
Great idea, thanks!
